sources/world/Makefile

```diff
@@ -1,38 +1,45 @@
 .DEFAULT_GOAL := ../../data/world_countries_v1.geo.json
 
 clean: ## Remove output files
-	rm -f ne_10m_admin_1_states_provinces.topo.json \
-  ../../data/admin_regions_lvl2_v1.geo.json \
+	rm -f ../../data/admin_regions_lvl2_v1.geo.json \
   ../../data/admin_regions_lvl2_v2.topo.json \
   ../../data/world_countries_v1.geo.json \
   ../../data/world_countries_v7.topo.json \
    /tmp/wikidata.csv /tmp/area.csv /tmp/population.csv \
-   /tmp/missing-area.csv /tmp/missing-population.csv
+   /tmp/missing-area.csv /tmp/missing-population.csv \
+   /tmp/ne_10m_admin_1_states_provinces.topo.json \
+   /tmp/additional-regions.geo.json
 
-ne_10m_admin_1_states_provinces.topo.json: ## Uncompress the NE regions dataset
+/tmp/ne_10m_admin_1_states_provinces.topo.json: ## Uncompress the NE regions dataset
 	gunzip --keep ./ne_10m_admin_1_states_provinces.topo.json.gz
+	mv ne_10m_admin_1_states_provinces.topo.json /tmp
 
+/tmp/ne_10m_admin_1_states_provinces.geo.json: /tmp/ne_10m_admin_1_states_provinces.topo.json
+	mapshaper -i /tmp/ne_10m_admin_1_states_provinces.topo.json \
+  -o format=geojson /tmp/ne_10m_admin_1_states_provinces.geo.json
 
-../../data/admin_regions_lvl2_v2.topo.json: ne_10m_admin_1_states_provinces.topo.json## Regions GeoJSON
-	mapshaper -i ne_10m_admin_1_states_provinces.topo.json \
-  -filter "['TW'].indexOf(iso_a2) > -1" \
-  -dissolve copy-fields="iso_3166_2,name,iso_a2,adm0_a3,admin" \
-  -each "iso_3166_2='CN-TW',name='Taiwan'" \
-  -o /tmp/china_states.shp
-	mapshaper -i combine-files ne_10m_admin_1_states_provinces.topo.json \
-  /tmp/china_states.shp \
+/tmp/additional-regions.geo.json: /tmp/ne_10m_admin_1_states_provinces.geo.json ./duplicated-regions.csv
+	cd ../../scripts && \
+  node duplicate-regions.js -v \
+    ../sources/world/duplicated-regions.csv \
+    /tmp/ne_10m_admin_1_states_provinces.geo.json \
+    /tmp/additional-regions.geo.json
+
+../../data/admin_regions_lvl2_v2.topo.json: /tmp/ne_10m_admin_1_states_provinces.topo.json /tmp/additional-regions.geo.json ## Regions GeoJSON
+	mapshaper -i   combine-files \
+  /tmp/ne_10m_admin_1_states_provinces.topo.json /tmp/additional-regions.geo.json \
   -merge-layers force \
+  -dissolve "iso_3166_2" copy-fields="iso_3166_2,name,iso_a2,adm0_a3,admin" \
```

Comment:
@nickpeihl this is the main part of my last change. I removed the ad-hoc management of Taiwan and moved all that logic to the `duplicated-regions.js` script that takes a CSV with the ISO codes that you want to extract.

Later on, the `admin_regions_lvl2_v2.topo.json` target takes the output and the original NE dataset and merges them, and as you did, there is a `-dissolve` for the regions to merge the origin regions that are a single region in the duplicated zone, as for Taiwan or Caribbean Netherlands.